app/assets/stylesheets/shared/header.scss

```diff
@@ -39,6 +39,7 @@
   }
 
   .dropdown-menu {
+    position: absolute;
```

Comment:
Bootstrap was doing a thing where if you were on a small screen, the positioning switched to `static` and that opened up the dropdown within itself. We want it to be positioned absolutely at all times.